libdnf5/base/transaction.cpp

```diff
@@ -981,7 +979,7 @@ bool Transaction::Impl::check_gpg_signatures() {
                         signature_problems.push_back(
                             err_msg + import_repo_keys_result_to_string(ImportRepoKeysResult::ALREADY_PRESENT));
                         result = false;
-                        continue;
+                        break;
```

Comment:
I'm not sure about all these `breaks`. Doesn't it mean that we will return to the user only the first signature problem instead of accumulating all of them?

Reply:
Oh, right. For larger transaction the new behaviour is definitely better because the root cause of the issue is more clearly visible.